fixtures/ai-app/tests/index.test.ts

```diff
@@ -20,18 +20,21 @@ describe("'wrangler dev' correctly renders pages", () => {
 		await stop?.();
 	});
 
-	it("ai binding is defined ", async ({ expect }) => {
+	it("ai binding methods ", async ({ expect }) => {
 		const response = await fetch(`http://${ip}:${port}/`);
 		const content = await response.json();
-		expect(content).toEqual({
-			binding: {
-				fetcher: {},
-				lastRequestId: null,
-				logs: [],
-				options: {},
-			},
-			fetch: "function",
-			run: "function",
+		expect((content as Record<string, object>).fetch).toEqual("function");
+		expect((content as Record<string, object>).run).toEqual("function");
+	});
+
+	it.skip("ai binding properties", async ({ expect }) => {
```

Comment:
Would be good to comment when this test can be unskipped

```suggestion
	// TODO: unskip when https://github.com/cloudflare/workerd/pull/2095 is merged and released
	it.skip("ai binding properties", async ({ expect }) => {
```